src/group/stabchain/mod.rs

```diff
@@ -87,6 +87,16 @@ where
         )
     }
 
+    /// Get the strong generating set of this stabiliser chain.
+    pub fn strong_generating_set(&self) -> Vec<P> {
```

Comment:
Just wondering, does every stabilizer chain automatically give a strong generating set? My understanding was that some algorithms do not construct a sgs directly in this manner.

Reply:
The union of all generating sets will always give a strong generating set. The definition is that G^(i) is equal to the group generated by the intersection of G^(i) and S (where S is the strong generating set) for each level i . If it includes the gens from each level, then it follows you'll get a SGS